Tabloid/Controllers/TagController.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Tabloid.Repositories;
+
+namespace Tabloid.Controllers
```

Comment:
Should this controller be auth protected? do we need any other auth checks on the route?